src/test/java/seedu/address/ui/HelpWindowTest.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.ui;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testfx.api.FxRobot;
+import org.testfx.framework.junit5.ApplicationExtension;
+import org.testfx.framework.junit5.Start;
+
+import javafx.application.Platform;
+import javafx.scene.input.KeyCode;
+import javafx.stage.Stage;
+
+@ExtendWith(ApplicationExtension.class)
+public class HelpWindowTest {
+
+    private HelpWindow helpWindow;
+
+    @Start
+    public void start(Stage stage) {
+        helpWindow = new HelpWindow(stage);
+        helpWindow.show();
+    }
+
+    @DisabledOnOs(OS.LINUX)
+    @Test
+    public void closeOnEscapeKeyPress(FxRobot robot) {
+        assertTrue(helpWindow.isShowing());
+        robot.press(KeyCode.ESCAPE);
+        assertFalse(helpWindow.isShowing());
```

Comment:
Is there a risk of a race condition here (the takes a while for the key press event to register)?

Reply:
Could be possible. I can add a `sleep` between line 33 and 34 for buffer.